### PR TITLE
Cards

### PR DESCRIPTION
**Pull request by adamruzicka**

### CODE REVIEW

File: src/PresentationalComponents/FilterList/FilterList.js [hunk 1]
```diff
@@ -135,23 +139,30 @@ export class FilterList extends Component {
...
                             <Checkbox id={ `app-check-${ app.id}` }
                                 data-event-type-id={ app.id }
                                 label={ app.attributes.name }
                                 aria-label={ app.attributes.name }
                                 onChange={ () => this.selectFilter('appIds', app.id) }
                                 defaultChecked={ this.state.selected.appIds[app.id]  } />
-                            { this.eventTypesList(app.eventTypes, app.id) }
-                        </CardBody>
+                        </CardHeader>
+                        { this.state.selected.appIds[app.id] &&
```
Comment (adamruzicka) [inline src/PresentationalComponents/FilterList/FilterList.js:127]: This works fine if I'm creating a new endpoint. If I'm editing one which already has some filter set, unchecking the application checkbox doesn't make the rest disappear, @bastilian any idea what's going on?

**Comment by bastilian** [comment]
If an app has no event-types should we still show the `RadioToggle`? It seems confusing and it might be better to not show it. The first and second app in this screenshot have no event_types in this example: 

Screen Shot 2019-04-01 at 10 11 03

**Comment by adamruzicka** [comment]
Fixed

image

**Comment by bastilian** [comment]
Awesome! Thanks @adamruzicka!

**Pull request merged**